wavefront/templates/proxy-deployment.yaml

```diff
@@ -37,6 +37,7 @@ spec:
           value: {{ .Values.proxy.args }}
           {{- if .Values.proxy.tracePort }} --traceListenerPorts {{ .Values.proxy.tracePort }}{{- end -}}
           {{- if .Values.proxy.jaegerPort }} --traceJaegerListenerPorts {{ .Values.proxy.jaegerPort }}{{- end -}}
+          {{- if .Values.proxy.traceJaegerHttpListenerPort }} --traceJaegerHttpListenerPort {{ .Values.proxy.traceJaegerHttpListenerPort }}{{- end -}}
```

Comment:
The proxy flag should still be `--traceJaegerHttpListenerPorts` (with an 's' at the end). But the helm property will be singular.